cfgov/v1/jinja2/v1/landing-page-1/index.html

```diff
@@ -12,7 +13,37 @@
     <div class="block
                 block__flush-top">
         {# TODO: Add main molecules and organisms. #}
-        {{ hero.render() }}
+        <div class="o-link-blob-group">
+            {{ link_blob.render({
+                'heading': 'This is an example of a Half Width Link Blob',
+                'content': 'To highlight navigational links and provide
+                            context about what users can expect on the linked
+                            page',
+                'links': [
+                    {
+                        'text': 'This is a Link List molecule inside the blob',
+                        'url': '/some-url/'
+                    },
+                    {
+                        'text': 'This is another Link List molecule inside the
+                                 blob',
+                        'url': '/some-other-url/'
+                    }
+                ]
+            }) }}
+            {{ link_blob.render({
+                'heading': 'This is another example of a Half Width Link Blob',
+                'content': '<strong>Example:</strong> About Us landing page,
```

Comment:
@KimberlyMunoz it's not in the spec, entirely my own opinion. If it doesn't make sense I can remove it.